Bootstrap.php

```diff
@@ -289,11 +289,11 @@ public static function validateMyAttribute(array $attribute)
     protected function createMyMenu()
     {
         if (version_compare(
-            Shopware::VERSION,
-            self::NEW_ENTITY_MANAGER_VERSION
-        ) < 0
+                Shopware::VERSION,
```

Comment:
Extra spaces were added by code formatter. It is because of 2 layers of bracket. 

if (version_compare(
        Shopware::VERSION,
        self::NEW_ENTITY_MANAGER_VERSION
    ) < 0
) {
    $parentMenu = $this->Menu()->findOneBy('id', self::MENU_PARENT_ID);
} else {